<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountCreationDialog</class>
 <widget class="QDialog" name="AccountCreationDialog">
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0"><widget class="QLabel" name="label_user"><property name="text"><string>Username</string></property></widget></item>
   <item row="0" column="1"><widget class="QLineEdit" name="usernameEdit"/></item>
   <item row="1" column="0"><widget class="QLabel" name="label_type"><property name="text"><string>Account Type</string></property></widget></item>
   <item row="1" column="1"><widget class="QComboBox" name="accountTypeCombo">
     <item><property name="text"><string>Main</string></property></item>
     <item><property name="text"><string>Savings</string></property></item>
     <item><property name="text"><string>QarzolHasaneh</string></property></item>
   </widget></item>
   <item row="2" column="0"><widget class="QLabel" name="label_balance"><property name="text"><string>Initial Balance</string></property></widget></item>
   <item row="2" column="1"><widget class="QLineEdit" name="balanceEdit"/></item>
   <item row="3" column="1"><widget class="QPushButton" name="createBtn"><property name="text"><string>Create</string></property></widget></item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>